fili-core/src/main/java/com/yahoo/bard/webservice/web/responseprocessors/MappingResponseProcessor.java

```diff
@@ -107,14 +110,32 @@ public HttpErrorCallback getStandardError(
         return new HttpErrorCallback() {
             @Override
             public void invoke(int statusCode, String reason, String responseBody) {
-                LOG.error(ErrorMessageFormat.ERROR_FROM_DRUID.logFormat(responseBody, statusCode, reason, druidQuery));
+                String druidQueryString;
+                ObjectWriter druidQuerySerializer = getObjectMappers().getMapper().writer();
+                try {
+                    druidQueryString = druidQuerySerializer.writeValueAsString(druidQuery);
+                } catch (JsonProcessingException jse) {
+                    try {
+                        druidQueryString = druidQuery.toString();
+                    } catch (Exception e) {
+                        LOG.warn("Error invoking a druid query's toString.", e);
+                        druidQueryString = "QUERY'S `toString` FAILED";
+                    }
+                    LOG.warn(String.format("Failed to serialize druid query %s", druidQueryString), jse);
```

Comment:
This isn't supposed to be here. I think I noticed that we weren't serializing the druid query in this log message, and added code to do so, but then was all "Meh, not worth another PR, thanks to the query id showing up in every log message." I must not have checked out a fresh master before adding the query id to the errror message.